### DISCUSSION TITLE
Autorefresh of drawings from a .py file

### DISCUSSION THREAD

**Discussion by fbob** [spakin/SimpInkScr#111] [Ideas]
Hi,

SimpInkScr is really a great extension, many thanks ! 

The autorefreshing of the drawing according to the modifications of the .py file could be a great feature which could help the workflow a lot.
 
Or could it be possible to build the .py script inside vscode for example, generate and update the svg each time the .py file is modified and see the result inside "vscode" or another image viewer accordingly ? 

Anyway many thanks for your great extension !

**Comment by spakin** [comment]
That indeed would be nice, but I lack the VSCode expertise to implement it.  I added the "help wanted" label to this issue in case someone reading this would like to contribute an auto-refresh feature.

**Reply by fbob** [reply]
Thanks !
